modules/remotecontrol/Receiver.js

```diff
@@ -39,9 +58,7 @@ class Receiver {
      * @param {Object} event the remote control event.
      */
     _onRemoteControlEvent(participant, event) {
-        if(event.type === "remote-control-event")
+        if(event.type === API_EVENT_TYPE && this.enabled)
```

Comment:
It would be good to log some warning here for debugging purposes if the method is called when the Receiver is not enabled.